app/templates/main/browse-series.html

```diff
@@ -334,7 +373,6 @@
             </div>
         </details>
     </div>
-    <!-- END MOBILE FILTERS -->
```

Comment:
do we want these block comments or not?

Reply:
I think the top block comment should suffice and they get differentiated with other block start comment